src/components/broadcast-box/BroadcastBox.jsx

```diff
@@ -1,77 +1,166 @@
+import { useState } from "react";
 import classNames from "classnames";
+import { Box } from "@mui/material";
 import styles from "./BroadcastBox.module.css";
-import { Box, Typography } from "@mui/material";
-import { AvatarGridBox, CustomIconButton } from "components";
+
+import {
+  AvatarGridBox,
+  BroadcastBoxHeader,
+  CustomIconButton,
+  OptionsMenu,
+  ReplyDialog,
+} from "components";
 import {
   CommentIcon,
+  DeleteIcon,
+  EditIcon,
   EllipsisHorizontalIcon,
-  FilledBookmarkIcon,
-  FilledHeartIcon,
+  OutlinedBookmarkIcon,
+  OutlinedHeartIcon,
+  FollowAccountIcon,
   HimanshuJPG,
+  UnfollowAccountIcon,
 } from "assets";
 
+// these variables are for design purpose only, they will get removed
+
 const post =
   "Hey 👋 everyone\n\nfinally, I have completed all the basic features of the marvelsQuiz app\n\n🌐Live URL: https://marvelsquiz.vercel.app\n🔗GitHub link: https://github.com/hsnice16/react-marvelsQuiz\n\nfeedbacks are appreciated.";
 
+const options = [
+  {
+    _id: "001",
+    color: "normal",
+    item: (
+      <>
+        <FollowAccountIcon /> Follow @hsnice16
+      </>
+    ),
+  },
+  {
+    _id: "002",
+    color: "normal",
+    item: (
+      <>
+        <UnfollowAccountIcon /> Unfollow @hsnice16
+      </>
+    ),
+  },
+  {
+    _id: "003",
+    color: "error",
+    item: (
+      <>
+        <DeleteIcon /> Delete this Broadcast
+      </>
+    ),
+  },
+  {
+    _id: "004",
+    color: "normal",
+    item: (
+      <>
+        <EditIcon /> Edit this Broadcast
+      </>
+    ),
+  },
+];
+
 export const BroadcastBox = () => {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [openReplyDialog, setOpenReplyDialog] = useState(false);
+
+  const handleMenuIconButtonClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleReplyClick = () => {
+    setOpenReplyDialog(true);
+  };
+
   return (
-    <AvatarGridBox
-      imgSrc={HimanshuJPG}
-      imgAlt="Himanshu Avatar"
-      className={styles.avatarBox}
-    >
-      <Box className={styles.container}>
-        <Box className={styles.header}>
-          <Box className={styles.name}>
-            <Typography component="h2">Himanshu Singh</Typography>
-            <Typography component="p">@hsnice16</Typography>
-          </Box>
+    <>
+      <ReplyDialog
+        openReplyDialog={openReplyDialog}
+        setOpenReplyDialog={setOpenReplyDialog}
+      />
 
-          <CustomIconButton ariaLabel="menu" className={styles.btnMenu_icon}>
-            <EllipsisHorizontalIcon />
-          </CustomIconButton>
-        </Box>
+      <AvatarGridBox
+        imgSrc={HimanshuJPG}
+        imgAlt="Himanshu Avatar"
```

Comment:
Here for alt in image, if you have a property or something, put that so that its different for different people if the image doesn't load.

Reply:
Thanks for reviewing, Hamza

Yeah, here `imgAlt` is a prop. In the component, I'm assigning its value to `alt` of the image.